test/test_helper.rb

```diff
@@ -47,6 +47,7 @@ def before_setup
     ShopifyApp::InMemorySessionStore.clear
     ShopifyAppConfigurer.call
     Rails.application.reload_routes!
+    ShopifyApp.configuration.log_level = :warn
```

Comment:
I had to set a log_level for our tests. I decided on `warn` instead of off because some tests are expecting to see some logs come through. All of those tests are rescued errors so I figured `:warn` was the best level. LMK what you think